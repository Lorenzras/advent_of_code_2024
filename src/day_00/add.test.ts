import { add } from '.';

describe('testing add file', () => {
  test('empty string should result in zero', () => {
    expect(add('')).toBe(0);
  });
  test('"1,2,3" should result in 6 ', () => {
    expect(add('1,2,3')).toBe(6);
  });
  test('negative number should throw error', () => {
    expect(() => add('1,-2,3')).toThrowErrorMatchingSnapshot();
  });
});