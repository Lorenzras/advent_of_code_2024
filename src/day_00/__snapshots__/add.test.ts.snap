// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing add file negative number should throw error 1`] = `"Negatives are not allowed: -2"`;
